draft-song-dns-wireformat-http-01.txt

```diff
@@ -101,18 +102,20 @@ Table of Contents
    [I-D.mohan-dns-query-xml] encoding for DNS data, in this approach
    wire-format data is wrapped with a HTTP header and transmitted on
    port 80 or 443.  This protocol is not designed with web developers or
-   advanced usage in application development in mind.  It simply serves
-   as a sort of DNS VPN, and does not introduce another format of DNS
-   data.
+   advanced usage in application development in mind (although it is
```

Comment:
I would like to soften or broaden the scope of targeted audience here, so that we can win more supporter in this community. How about  "This protocol is mainly used as a DNS tunnel mechanism as most tunnel technology dose. It is also potential to provide new DNS APIs for advanced usage in application development. For example, web developers can create arbitrary HTTP/HTTPS queries and get DNS responses in their JavaScript apps"